docs/tables/apache_access_log/index.md

```diff
@@ -230,6 +230,7 @@ partition "apache_access_log" "s3_logs" {
     connection  = connection.aws.logging
     bucket      = "apache-access-logs"
     prefix      = "logs/"
+    file_layout = `%{YEAR:year}/%{MONTHNUM:month}/%{MONTHDAY:day}/%{DATA}.log`
```

Comment:
```suggestion
```

To simplify the example, maybe we should take this out? It feels like I need to have `file_layout` and need to use this directory structure based on the example. I suspect most people will use `prefix`.